Organization/Employee+CoreDataProperties.h

```diff
@@ -0,0 +1,17 @@
+#import "Employee+CoreDataClass.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Employee (CoreDataProperties)
+
++ (NSFetchRequest<Employee *> *)fetchRequest;
+
+@property (nullable, nonatomic, copy) NSString *firstName;
+@property (nullable, nonatomic, copy) NSString *lastName;
+@property (nonatomic) int32_t salary;
+@property (nullable, nonatomic, copy) NSString *fullName;
```

Comment:
Full name should not be stored in database, it should be calculated dynamically in runtime.